doc/getting-started/building-from-source.rst

```diff
@@ -312,10 +312,23 @@ If you only need the hypervisor, then use this command:
    $ make -C hypervisor
    $ make -C misc/efi-stub HV_OBJDIR=$PWD/hypervisor/build EFI_OBJDIR=$PWD/hypervisor/build
 
-The acrn.efi will be generated in directory: . /hypervisor/build/acrn.efi
+The``acrn.efi`` will be generated in directory: ``./hypervisor/build/acrn.efi``
 (`Slim bootloader
 <https://www.intel.com/content/www/us/en/design/products-and-solutions/technologies/slim-bootloader/overview.html>`_)
 hypervisor.
 
+As mentioned in :ref:`ACRN Configuration Tool <vm_config_workflow>`,
+Board configuration and VM configuration could be imported from XML files.
+If you want to build hypervisor with XML configuration files, please specify the
+file location as follows:
+
+.. code-block:: none
+
+   $ BOARD_FILE=/home/acrn-hypervisor/misc/acrn-config/xmls/board-xmls/apl-up2.xml 
```

Comment:
I know I asked for a real example but upon reading it, I think this line would be sufficiently clear:
```suggestion
   $ make BOARD_FILE=/path/to/$(BOARD).xml SCENARIO_FILE=/path/to/$(SCENARIO).xml FIRMWARE=uefi
```